phoenix-core/src/it/java/org/apache/phoenix/end2end/index/MutableIndexReplicationIT.java

```diff
@@ -42,11 +43,7 @@
 import org.apache.hadoop.hbase.HConstants;
 import org.apache.hadoop.hbase.HTableDescriptor;
 import org.apache.hadoop.hbase.TableName;
-import org.apache.hadoop.hbase.client.HBaseAdmin;
-import org.apache.hadoop.hbase.client.HTable;
-import org.apache.hadoop.hbase.client.Result;
-import org.apache.hadoop.hbase.client.ResultScanner;
-import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.client.*;
```

Comment:
I think we want to keep the import expanded.

Reply:
Reverted back to expanded import